packages/frontend/components/student-journey/step-title.js

```diff
@@ -8,7 +8,7 @@ function StepTitle({ step, title, children, fullWidth, className }) {
       <h4 className="font-bold">STEP {step}</h4>
       <Title
         className="mb-4 w-6/12 uc-onboarding-journey-step-title"
-        type="small"
+        variant="xs"
```

Comment:
Maybe I'm missing the context but not quite catch why this is related to testing? 😕

Reply:
Nope, you aren't missing the context. There is no relation between testing and these changes. I saw that when I write tests there was no standard for these keys. I decide to change all "type" props with "variant" because the type can be standard html prop sometimes, like in buttons. I can send new PR if needed.